modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/persistence/test/JournalArticleFinderTest.java

```diff
@@ -194,6 +196,73 @@ public void testDraftArticles() throws Exception {
 			JournalArticleConstants.CLASS_NAME_ID_DEFAULT, queryDefinition, 0);
 	}
 
+	@Test
+	public void testFindByG_A_LtDD_GtED_ST() throws Exception {
+		JournalArticle article = JournalTestUtil.addArticle(
+			_group.getGroupId(), 0);
+
+		QueryDefinition<JournalArticle> queryDefinition = new QueryDefinition<>(
+			WorkflowConstants.STATUS_APPROVED, 0, 1, null);
+
+		article = _journalArticleFinder.filterFindByG_A_LtDD_GtED_ST(
+			_group.getGroupId(), article.getArticleId(), new Date(), new Date(),
+			WorkflowConstants.STATUS_APPROVED, queryDefinition
+		).get(
+			0
+		);
```

Comment:
@balazssk, we normally do this in 2 steps. first the filder find, then the get :sweat:

Reply:
I was surprised that SF let it through as well.